lobby-db/build.gradle

```diff
@@ -7,6 +7,7 @@ buildscript {
 
 plugins {
   id "org.flywaydb.flyway" version "5.0.7"
+  id 'java'
```

Comment:
I'm curious why the java plugin is needed

Reply:
it creates some defaults for the zip task. Specifically in this case we get placement in build folder instead of at project root. From google researching it seems not uncommon to include java just for the zip defaults.